app/models/concerns/status_extension.rb

```diff
@@ -0,0 +1,18 @@
+module StatusExtension
+  extend ActiveSupport::Concern
+  included do
+    after_commit :update_index
+  end
+
+  def update_index
+    data = [{
+      layer_availability_score_f: { set: layer.availability_score },
+      layer_slug_s: layer.slug
+    }]
+    Blacklight.default_index.connection.update(
+      params: { commitWithin: 500, overwrite: true },
```

Comment:
I would either choose a larger commitWithin (say 5 seconds) or make this configurable. I don't think we need a fast commit for the availability_score, right?

Reply:
Nope not at all. I'll make this configurable with a larger default